src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -170,6 +172,11 @@ private boolean initSparks() {
 
     errors += SparkMaxUtils.check(elevatorRight.follow(elevatorLeft, true));
 
+    //inverting stuff
+    errors += SparkMaxUtils.check(armEncoder.setInverted(true));
+    errors += SparkMaxUtils.check(elevatorLeftEncoder.setInverted(true));
```

Comment:
Are these just for the absolute encoders or for the motors also? Because the absolute encoders would be `elevatorLeftAbsEncoder ` and `armAbsoluteEncoder `. These are relative encoders (the ones built in to the motors)

Reply:
Yep, you are correct. They are for the absolute encoders.